pkg/contracts/src/RegistryCommunityV0_0.sol

```diff
@@ -0,0 +1,613 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.19;
+
+import "forge-std/console.sol";
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {ERC165Checker} from "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+
+import {OwnableUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import {ReentrancyGuardUpgradeable} from
+    "openzeppelin-contracts-upgradeable/contracts/security/ReentrancyGuardUpgradeable.sol";
+import {AccessControlUpgradeable} from
+    "openzeppelin-contracts-upgradeable/contracts/access/AccessControlUpgradeable.sol";
+
+import {IAllo} from "allo-v2-contracts/core/interfaces/IAllo.sol";
+import {Clone} from "allo-v2-contracts/core/libraries/Clone.sol";
+import {IRegistry, Metadata} from "allo-v2-contracts/core/interfaces/IRegistry.sol";
+import {FAllo} from "./interfaces/FAllo.sol";
+import {ISafe} from "./interfaces/ISafe.sol";
+import {RegistryFactory} from "./RegistryFactory.sol";
+import {IPointStrategy, CVStrategy, StrategyStruct} from "./CVStrategy.sol";
+
+contract RegistryCommunityV0_0 is
+    OwnableUpgradeable,
+    UUPSUpgradeable,
+    ReentrancyGuardUpgradeable,
+    AccessControlUpgradeable
```

Comment:
Oh okok nice so we have a role managament already with AccessControlUpgreadeable